model/articles.models.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../db/connection')
+
+
+exports.fetchArticlebyId = (id) => {
+    return db.query('SELECT * FROM articles WHERE article_id = $1', [id])
+    .then((response) => {
+        if (response.rows.length === 0) {
+           return Promise.reject({ status: 404, msg: 'Not Found'})
+        } else {
+        return response.rows[0]
+        }
+    })
+}
+
+
+exports.updateArticleById = (id, updateVoteBy) => {
+    return db.query(
+    `UPDATE articles
+    SET votes = $1
+    WHERE article_id = $2
```

Comment:
Here you are setting the value of votes to whatever the value of updateVoteBy is. However, this will not allow you to add to existing votes in the database. Have a look at whether you can perform mathematical equations in the SET part of your query